### PR TITLE
[4.0] com_mails: string typo & template fields labels display missing

### PR DESCRIPTION
**Pull request by infograf768**
Pull Request for Issue https://github.com/joomla/joomla-cms/pull/28722#issuecomment-633881966

### Summary of Changes
1. This corrects the language string by taking off `when` to get
`COM_USERS_MAIL_REMINDER_DESC="This mail is sent to a user by the \"Forgot your username?\" link e.g. in a login form."` Can be seen on review.

2. It also corrects the template edit missing labels for the switchers.

### Testing Instructions
Display for example index.php?option=com_mails&view=template&layout=edit&template_id=com_config.test_mail&language=fr-FR [Choose any language]

### Before patch
Screen Shot 2020-05-26 at 11 04 35

### After patch
Screen Shot 2020-05-26 at 10 57 02

### CODE REVIEW

**Comment by toivo** [comment]
I have tested this item :white_check_mark: successfully on ab7d7233211a58b7c6231116b868c33adcb46180<br /><br />Tested successfully in Nightly Build of 26 May.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29224">issues.joomla.org/tracker/joomla-cms/29224</a>.</sub>

File: administrator/language/en-GB/com_users.ini [hunk 1]
```diff
@@ -188,7 +188,7 @@ COM_USERS_MAIL_PLEASE_FILL_IN_THE_FORM_CORRECTLY="Please fill in the form correc
 COM_USERS_MAIL_PLEASE_FILL_IN_THE_MESSAGE="Please enter a message"
 COM_USERS_MAIL_PLEASE_FILL_IN_THE_SUBJECT="Please enter a subject"
 COM_USERS_MAIL_PLEASE_SELECT_A_GROUP="Please select a Group"
-COM_USERS_MAIL_REMINDER_DESC="This mail is sent to a user when by the \"Forgot your username?\" link e.g. in a login form."
+COM_USERS_MAIL_REMINDER_DESC="This mail is sent to a user by the \"Forgot your username?\" link e.g. in a login form."
```
Comment (richard67) [inline administrator/language/en-GB/com_users.ini]:
```suggestion
COM_USERS_MAIL_REMINDER_DESC="This mail is sent to a user by using the \"Forgot your username?\" link e.g. in a login form."
```
To make it consistent with COM_USERS_MAIL_PASSWORD_RESET_DESC.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 409acffaa08cf373f9c3c48cb1cf5860d9d7d8fa<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29224">issues.joomla.org/tracker/joomla-cms/29224</a>.</sub>

**Comment by infograf768** [comment]
rtc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29224">issues.joomla.org/tracker/joomla-cms/29224</a>.</sub>

**Comment by richard67** [comment]
Restored @toivo 's test result because last change was only a small correction of the language string.

**Pull request merged**